package.json

```diff
@@ -68,7 +68,7 @@
     "webpack-notifier": "^1.2.1"
   },
   "dependencies": {
-    "bluebird": "^3.0.5",
+    "bluebird": "^3.1.1",
```

Comment:
This shouldn't be here (or if you really wanna update bluebird, you also need to update shrinkwrap, but I prefer you do it separately)